CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ## [unreleased] (Added 🚀 | Changed | Removed 🗑 | Fixed 🐞 | Chore 👨‍💻 👩‍💻)
 
+### Chore 👨‍💻 👩‍💻
+
+-   Remove `nodeSearch.service.ts` and `searchedNodePaths` from store as they can be derived from `searchPattern`.
```

Comment:
Shouldn't that be under Removed?
And also there should be a link to the PR (or Issue)

Reply:
Removed is for a removed Feature. For the user everything behaves the same. It is an internal (chore) change.

But I added a link to the PR